server/src/main/resources/db/migration/V1_16__Admin_Statistics.sql

```diff
@@ -0,0 +1,130 @@
+CREATE TABLE public.admin_statistics(
```

Comment:
As this schema migration is rather complex, could you document the general idea and the components (new tables etc.) that contribute to that idea?